include/rocksdb/db.h

```diff
@@ -1140,6 +1140,10 @@ class DB {
   virtual Status GetCurrentWalFile(
       std::unique_ptr<LogFile>* current_log_file) = 0;
 
+  // Retrieves the creation time of the oldest file in the DB.
+  // This API only works if max_open_files = -1;
```

Comment:
Explain what `creation_time = 0` means here.

Reply:
May not be relevant to talk about creation_time = 0 any more as the behavior is to just return the file with lowest timestamp and it could be 0 depending on the Env's behavior.